core/src/main/java/io/spine/core/CommandEnvelope.java

```diff
@@ -113,13 +114,20 @@ public ActorContext getActorContext() {
     }
 
     /**
-     * Sets the context of the enclosed command as the origin for the event being built.
+     * Passes data from the enclosed command to the event context being built.
      *
-     * @param builder event context builder into which set the event origin context
+     * <p>In particular: 
+     * <ul>
+     *     <li>the command identifier is set as the root command identifier;</li>
+     *     <li>the command origin is set to the context of the enclosed event.</li>
+     * </ul>
+     * 
+     * @param builder event context builder which is filled with the enclosed message data
      */
     @Override
-    public void setOriginContext(EventContext.Builder builder) {
+    public void passToEventContext(EventContext.Builder builder) {
```

Comment:
I am not sure about the new name. It doesn't sound right to be, but I cannot suggest better alternatives. What options did you consider?